tests/common.py

```diff
@@ -114,6 +114,9 @@ def stop_hass():
 def async_test_home_assistant(loop):
     """Return a Home Assistant object pointing at test config dir."""
     hass = ha.HomeAssistant(loop)
+    hass.config_entries = config_entries.ConfigEntries(hass, {})
```

Comment:
Is there a way we can initialize this with the normal core code path? It feels like some smell that we have to manually create this here.

Reply:
So this is getting set by `homeassistant.bootstrap#from_config_dict`. We also hardcode the core config that is loaded in that method.

Reply:
I am not 100% sure what would be the best approach here. I wouldn't want to move it to core as it is related to components and `core.py` does nothing with components (well except for tracking loaded ones in `hass.config.components`).